docs/dev/plugins/cloudformation/cloudformation-basics.md

```diff
@@ -0,0 +1,36 @@
+# AWS CloudFormation
+How to transform a TOSCA model to an Amazon Web Services CloudFormation template.
+
+## Mapping
+see [this](mapping.md)
+
+## Transformation Lyfecycle
+
+### validate
+
+### prepare
+- Add missing nodes to fulfill requirements.
+- Split the graph, each part representing a AWS Resource
+
+### transform
+- transform everything into a CloudFormation JSON/YAML template
+
+### cleanup
+
+
+## Deployment / Readme
+- target artifact should be a single template file (JSON/YAML)
```

Comment:
If you already know which format you will use, you could write a ADR and mention it here. If not, forget my comment.